docs/audits/deprecated-api-usage.md

```diff
@@ -0,0 +1,17 @@
+# Avoid using deprecated GPT APIs
```

Comment:
Does this match the audit failure title?

Reply:
Good call, updated to match it.